tests/global-setup.ts

```diff
@@ -2,7 +2,7 @@ import { expect, chromium, FullConfig } from '@playwright/test';
 // Import Route directly so that other imports are not bundled in
 import { Route } from '../frontend/src/lib/utils/navigation/route';
 import candidateAppTranslations from '../frontend/src/lib/i18n/translations/en/candidateApp.json';
-import mockUser from '../backend/vaa-strapi/src/functions/mockData/mockUser.json';
+import mockUsers from '../backend/vaa-strapi/src/functions/mockData/mockUser.json';
```

Comment:
Can we also do the same thing as in `candidate.spec` ie. assing `const mockUser = mockUsers[1];` here so we don't get confused with the indices below